vignettes/advanced.Rmd

```diff
@@ -0,0 +1,128 @@
+---
+title: "Advanced usage"
+author: "Patrick Schratz, Kirill Müller"
+date: "`r Sys.Date()`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Kirill Müller}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+# Deployment
+
+## pkgdown deployment
```

Comment:
This section seems to discuss deployment to a `gh-pages` branch. Maybe add a discussion about the options, and how to use them with _tic_?

Do we have an example for deploying to a different target (AWS, ...)?

Reply:
Extended the section. Maybe there is already TMI in the advanced page already? Let's see later.